### ISSUE TITLE
Issue with derivation and explicit interface implementation.

### ISSUE THREAD

**Issue by hardhub** [bridgedotnet/Bridge#3432]
For some reason it says that the TestTime is string.

### Steps To Reproduce

https://dev.deck.net/04f893c4a32fff4b07abde9ae7f9320a
https://dotnetfiddle.net/1OM6St

```csharp
public interface ISome1
{
    TimeSpan? TestTime { get; }
}

public class Some1 : ISome1
{
   public string TestTime { get; set; }
   
   TimeSpan? ISome1.TestTime
   {
       get { return this.Parse(this.TestTime); }
   }
	
    private TimeSpan? Parse(string text)
    {
        if (text == null)
            return null;

        string[] values = text.Split(':');

        if (values.Length == 2)
            return new TimeSpan(0, 0, int.Parse(values[0]), int.Parse(values[1]));

        if (values.Length == 3)
            return new TimeSpan(0, int.Parse(values[0]), int.Parse(values[1]), int.Parse(values[2]));

        throw new FormatException("Unsupported format of TimeSpan.");
    }
}

public interface ISome2 : ISome1
{
    string Additional { get; }
}

public class Some2 : Some1, ISome2
{
   public string Additional { get; set; }
}

public class Program
{
    public static void Main()
    {
        ISome1 some2 = new Some2() { TestTime = "00:02:00" };

        Console.WriteLine(some2.TestTime?.Days);
    }
}

```

### Expected Result

```
0
```

### Actual Result

```
System.Exception: Main@https://dev.deck.net/RunHandler.ashx?h=-1696014459:16:103
init/</<@https://dev.deck.net/resources/js/bridge/bridge.min.js?16.8.1-201802200331:7:40146
i@https://dev.deck.net/resources/js/bridge/bridge.min.js?16.8.1-201802200331:7:5505
i@https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.4/jquery.min.js:2:27146
add@https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.4/jquery.min.js:2:27450
n.fn.ready@https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.4/jquery.min.js:2:29497
n.fn.init@https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.4/jquery.min.js:2:24943
n@https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.4/jquery.min.js:2:405
ready@https://dev.deck.net/resources/js/bridge/bridge.min.js?16.8.1-201802200331:7:5554
init/<@https://dev.deck.net/resources/js/bridge/bridge.min.js?16.8.1-201802200331:7:40122
init@https://dev.deck.net/resources/js/bridge/bridge.min.js?16.8.1-201802200331:7:40108
Bridge.assembly@https://dev.deck.net/resources/js/bridge/bridge.min.js?16.8.1-201802200331:7:40762
@https://dev.deck.net/RunHandler.ashx?h=-1696014459:4:1
```

**Comment by geoffreymcgill** [comment]
Thanks for reporting this issue. We are investigating. I'll keep this thread updated with our progress.

**Comment by fabriciomurta** [comment]
The test to reproduce the issue could be reduced to just an ordinary `int` variable, instead of a TimeSpan with cast and all: https://deck.net/12f8b0cf1442dfd4a724bf7dca32fcbd

**Comment by fabriciomurta** [comment]
The issue has been fixed and merged up the integration branch! It will make it to the next Bridge release (see attached milestone).

**Comment by hardhub** [comment]
Is it possible to download 16.8.2 nuget package as some kind of "early access"?

**Comment by geoffreymcgill** [comment]
Hi @hardhub,

Building the [dev](https://github.com/bridgedotnet/Bridge/tree/dev) branch in Release mode will get you the latest packages and is available anytime.

Hope this helps.

**Comment by hardhub** [comment]
Hi @geoffreymcgill 

I see... I did that some times.  
Although I mentioned nuget packages source where packages are available as Preview or Nightly builds.

**Comment by geoffreymcgill** [comment]
Fixed and included in the Bridge 16.8.2 release.
